README.md

```diff
@@ -825,6 +825,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 * [al8n/stretto](https://github.com/al8n/stretto) - A high performance thread-safe memory-bound Rust cache [![build badge](https://github.com/al8n/stretto/actions/workflows/ci.yml/badge.svg)](https://github.com/al8n/stretto/actions/workflows/ci.yml)
 * [jaemk/cached](https://github.com/jaemk/cached) — Simple function caching/memoization
 * [mozilla/sccache](https://github.com/mozilla/sccache/) - Shared Compilation Cache, great for Rust compilation [![build badge](https://api.travis-ci.org/mozilla/sccache.svg?branch=master)](https://travis-ci.org/mozilla/sccache)
+* [zkat/cacache-rs](https://github.com/zkat/cacache-rs) - A high-performance, concurrent, content-addressable disk cache, optimized for async APIs [![build badge](https://github.com/zkat/cacache-rs/workflows/CI/badge.svg)](https://github.com/zkat/cacache-rs/actions/workflows/ci.yml)
```

Comment:
It has more than 50 stars
![image](https://user-images.githubusercontent.com/36519478/221776056-f01f6099-ccea-45ac-a891-6160925b2ef7.png)
More than 2000 downloads
![image](https://user-images.githubusercontent.com/36519478/221776148-202a1abe-a266-408c-b857-a2c9c968bb38.png)

I liked the description hehe